internal/hcp/registry/json.go

```diff
@@ -93,7 +100,25 @@ func (h *JSONRegistry) CompleteBuild(
 	artifacts []sdkpacker.Artifact,
 	buildErr error,
 ) ([]sdkpacker.Artifact, error) {
-	return h.bucket.completeBuild(ctx, build.Name(), artifacts, buildErr)
+	name := build.Name()
```

Comment:
I had to look up to understand whose name was `name`
```suggestion
	buildName := build.Name()
```